modin/pandas/series_utils.py

```diff
@@ -331,7 +331,7 @@ def partition(self, sep=" ", expand=True):
             )
 
     def repeat(self, repeats):
-        return self._default_to_pandas(pandas.Series.str.repeat, repeats)
+        return Series(query_compiler=self._query_compiler.str_repeat(repeats))
```

Comment:
Thanks for doing this! What about the other cases `_default_to_pandas` is called in this file? Are there cases, beyond this file, where the decision to default to pandas is made before entering the query compiler layer?

Reply:
I think we should fix the other cases in this file and other places in another PR. Maybe someone added this case because the pandas and base query compilers already had `str_repeat`, but we just weren't using it.

Reply:
I would say we _shouldn't_ fix that particular issue in this PR, as it's unrelated to introducing a client QC